lib/Elastica/Type.php

```diff
@@ -299,6 +299,22 @@ public function setMapping($mapping)
     }
 
     /**
+     * Sets value type mapping for this type with timestamp .
+     *
+     * @param \Elastica\Type\Mapping|array $mapping Elastica\Type\MappingType object or property array with all mappings
+     *
+     * @return \Elastica\Response
+     */
+    public function setMappingWithTimeStamp($mapping,$timestamp)
```

Comment:
Yes, type should know about timestamp. But in type class there isn't any way of referencing mapping directly (on members). In setMapping method send is called immediately after making mapping, so i needed to set timestamp param before that. There is no room to update type w.r.t  timestamp.